docs/devlog/DAILY_DEVLOG_TEMPLATE.md

```
@@ -0,0 +1,28 @@
+
+## Next 24h Plan
+[What are you doing tomorrow?]
+
+## Links & Resources
+- GitHub Repository: [Link]
+- Demo Video: [Link]
+
+---
+I created this post for the purposes of entering the Gemini Live Agent Challenge.
+
+#GeminiLiveAgentChallenge #DevLog #BuildInPublic
```

markdownlint-cli2 (0.21.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)